application/ui/components/api-resource.jsx

```diff
@@ -51,13 +51,27 @@ module.exports = React.createClass({
 
         if (resource) {
             window.document.title = resource.title;
-            resourceComponent = (
+
+            // Allow the docs to use either methods or endpoints, but not both.
+            if (typeof(resource.endpoints) === 'undefined')
```

Comment:
Underscore has an `_.isUndefined()` method that we generally use for this since underscore is already included